include/zephyr/bluetooth/mesh/brg_cfg.h

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
According to IRL architecture discussion there should be separate bridge table module. These primitives should be part of its API.

Reply:
I needed a file to put some of these definition in and there is nothing preventing the bridge table module to use what ever of this.